### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by kay0u**
## Problem
- *Broken PDF viewer in shared folders*
- *Group and user are not associated*
- *The ldap admin page is not working*

## Solution
- *Fix https://github.com/YunoHost-Apps/nextcloud_ynh/issues/392*
- *Upgrade to 20.0.8*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [ ] **Code review** : 
- [ ] **Approval (LGTM)** : 
- [ ] **Approval (LGTM)** : 

## Package_check results
---
* An automatic package_check will be launch at https://ci-apps-dev.yunohost.org/, when you add a specific comment to your Pull Request: "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!"*

### CODE REVIEW

**Comment by kay0u** [comment]
!testme

File: manifest.json [hunk 1]
```diff
@@ -6,12 +6,12 @@
...
-    "version": "20.0.7~ynh1",
+    "version": "20.0.8~ynh1",
     "url": "https://nextcloud.com",
     "license": "AGPL-3.0",
     "maintainer": {
-        "name": "YunoHost Contributors",
-        "email": "apps@yunohost.org"
+        "name": "Kay0u",
+        "email": "pierre@kayou.io"
     },
```
Comment (alexAubin) [inline manifest.json:15]: :heart: :sweat_smile:

**Review by alexAubin — APPROVED** [review]
Naively LGTM but i'm not super familiar with those technicalities, or the whole LDAP group story

**Review by yalh76 — APPROVED** [review]
Quick review, but looks good to me

**Pull request merged**